models/brats_mri_segmentation/docs/README.md

```diff
@@ -94,3 +94,6 @@ This is an example, not to be used for diagnostic purposes.
 # References
 
 [1] Myronenko, Andriy. "3D MRI brain tumor segmentation using autoencoder regularization." International MICCAI Brainlesion Workshop. Springer, Cham, 2018. https://arxiv.org/abs/1810.11654.
+
+# License
+This model is released under Apache 2.0 terms. The license file is included with the model.
```

Comment:
I think for the Apache license, we should use the recommended way of applying it, for example:
https://github.com/Project-MONAI/model-zoo/blob/71e469723658f1be579423c328a85f7ce97cc52c/models/lung_nodule_ct_detection/LICENSE#L178-L187

Reply:
Thanks @wyli . I modified the license file. Could you please help to re-review it?

Reply:
sorry I didn't make it clear. I think we should include the full text of the license as file `models/model_name/LICENSE`. Additionally, for the rest of the files in folder `models/model_name` whenever we mention the license we use this paragraph:
https://github.com/Project-MONAI/model-zoo/blob/71e469723658f1be579423c328a85f7ce97cc52c/models/lung_nodule_ct_detection/LICENSE#L189-L201

Reply:
Thanks Wenqi, I updated the LICENSE files and readme.